site_scons/options.py

```diff
@@ -17,6 +17,22 @@
         help="""How many clonal families should we process per unseeded sample and locus? Defaults to 20,
         unless this is a test run in which case it defaults to 3.""")
 
+Script.AddOption('--only-seeds',
+        dest='only_seeds',
+        metavar='LIST',
+        default=None,
+        help="""Specify ':' separated list of partis seeds to process only those seed containing clusters.""")
+
+Script.AddOption('--match-indels-in-uid',
+        dest='match_indels_in_uid',
+        default=None,
+        help="""process only sequences matching the one indel in the sequence corresponding to the uid passed here in the annotation chosen in choose_cluster()""")
```

Comment:
@eharkins What is `choose_cluster()` here? Does this have to be used in conjunction with `--only-seeds`, or is it expected to? Would be nice to have a little more context here. Seems like this option would only ever apply towards a single specific seed/cluster. Maybe this should be specified on a seed by seed (or somehow generally cluster by cluster) basis in the input yaml for cft? Doesn't feel like a setting you'd build an entire dataset with.

Reply:
@metasoarous `choose_cluster()` refers to [the bin/process_partis.py function with that name](https://github.com/matsengrp/cft/blob/master/bin/process_partis.py#L272), which should be more clear here. Thanks!

re: 
> Maybe this should be specified on a seed by seed (or somehow generally cluster by cluster) basis in the input yaml for cft?

@psathyrella argues 
> I don't think the info.yaml is the place to do it -- that file i think should stay as just a record of what partis stuff was run and exists

What about just restricting both `--match-indels-in-uid` and `--ignore-seed-indels` to be used only when `--only-seeds` is used, since as you point out here they don't really make sense on the whole dataset level.

The only other way I could think of to associate the indel matching options with a particular seed would be to also pass (in another option) the seed id of the cluster to which we want to apply the indel matching options. This would be like using  `--only-seeds` to apply those those options to only a particular seed cluster, except it would allow you to process the rest of the dataset without those options. That use case seems less likely than just wanting to process one seed cluster with some special options.

Reply:
OK; Then what happens when the user wants to run a different seed with a different indel pattern? Do you have to run two separate scons commands? What happens when you come back later and just want to build the unseeded clusters, and so leave off these options, but then accidentally nuke your carefully crafted indel-pattern -aware builds for those particular seeds?

I don't like specifying detailed information like this at the command line for scons commands, and generally prefer that things be read in from a file whenever possible. Would @psathyrella be happy if we put this data in a separate input file? Maybe we could just copy over his yaml files to our own `datasets` dir, and add our CFT specific attributes there?

Reply:
yeah I don't care if cft gets the info from a file or from the command line, I just care that the info.yaml files in `/fh/fast/matsen_e/processed_data/partis` remain only a record of the partis runs, and not getting modified every time we want to rerun cft with different seed/indel options. So copying that info.yaml and modifying it seems sensible.

Reply:
>  Do you have to run two separate scons commands

Yes, since these options should be
> used only when --only-seeds is used, since as you point out here they don't really make sense on the whole dataset level.

This 
> accidentally nuke your carefully crafted indel-pattern -aware builds for those particular seeds

wouldn't happen because of [dataset tags](https://github.com/matsengrp/cft/pull/284/files/4049d7860dbd554998a7bbb07c57c81260942585#diff-fe51b584d3baf130b22e09b4ead595a7R94):

Despite the fact that it could work to specify things from the command line, the following makes sense:
> I don't like specifying detailed information like this at the command line for scons commands, and generally prefer that things be read in from a file whenever possible. 

Adding this info to a local copy of info.yaml files sounds good, is it worth setting up some script to (copy from `/fh/fast/matsen_e/processed_data/partis` and) add fields to particular seeds using options like the ones I added to site_scons/options.py, or should this be done manually?

Reply:
We agree; You could make this happen with separate commands going out into separate datasets using dataset tags. And if that's how you want to go, that's fine. Whether or not you do might depend on whether or not you want to be able to think about the modified analyses as the only ones, as apposed to an alternate version in some other data set. Separate datasets might confuse someone who didn't realize that they should really be looking at this "fixed" version over in another dataset. Then again, it's useful to be able to compare the with/without treatment. So maybe being able to do it both ways is important (or at least valuable)?